lib/read_table_A.pl

```diff
@@ -1901,7 +1901,39 @@ sub read_table_A {
         $object = '';
     }
     #-------------- End AoGSmartHome Objects ----------------
-
+    #-------------- MQTT Objects -----------------
+    elsif ( $type eq "MQTT_GATEWAY" ) {
```

Comment:
I would replace `MQTT_GATEWAY` with `MQTT_BROKER` here as that is the terminology that is used in MQTT-speak.